backend/django/create-model-and-api.mo.md

```diff
@@ -0,0 +1,219 @@
+# [MO] Create a model and an api route (~20 mins)
+
+## Owner: Alice Breton
+
+## Prerequisites (~2 mins)
+
+- Have a django project installed, [see this](https://github.com/bamlab/dev-standards/blob/master/backend/django/getting-started.mo.md).
+- Have an app (in this example Publications).
+- In this example we will use an already existing model 'city' in a 'locations' app, to show how to integrate other models with foreign keys.
+
+## Steps  
+
+- In this tutorial we will create a News model, add it to the Django admin back-office and an API route to get all the pieces of news that have been created.
+- To give a full example, we will ==== REMPLIR
+
+### Create a News model *(~10 min)*
+
+- In this example we will create a new model called News, with different attributes as you will see bellow. In the models.py file of your app (in our case Publications) create a new model. 
+
+ourproject/publications/models.py
+```python
```

Comment:
You can put the file path in comment in the python code quote :

```python
# File: "our_django_project/publications/models.py"

from django.db import models
```

Reply:
Good idea, done :)